third-party/README

```diff
@@ -9,7 +9,8 @@ Chapel itself.  Current subdirectories include:
 
 chpl-venv/
   Summary: Directory where several python packages are downloaded and
-           installed. Used for chpldoc and the testing system. See
+           installed. Used for chpldoc, generating standard module
+           documentation, testing system, and man page generation. See
```

Comment:
There's some weird grammatical tense stuff going on here.  I'd rearrange this to "Used by chpldoc and the testing system, and to generate the man page and standard module documentation."